stories/assets/js/init-loader.js

```diff
@@ -22,124 +22,147 @@ export const initializeComponents = (Story, context) => {
     // Initialize based on component type
     switch (componentType) {
         case 'language-switcher':
-            if (typeof window.UNDP?.langSwitch === 'function') {
-            window.UNDP.langSwitch();
-            } else {
+            // if (typeof window.UNDP?.langSwitch === 'function') {
```

Comment:
@mihir880 , your original idea was great - to check if the module is already loaded and use it that way, opting for load dynamically only if the function is not present. I'd rather keep it that way to make script more versatile. The only issue you have to resolve is changing `window.UNDP.functionName` in the condition to simply `functionName` as it is defined in the global scope if loaded as a separate script